lib/solr_handler.rb

```diff
@@ -22,8 +22,11 @@ def setup_raw_request_with_basic_auth request_context
 
 class SolrHandler
   def self.send_docs_to_solr(solr_url, docs)
-    solr = RSolr.connect :url => solr_url
-    solr.add docs
-    solr.commit
+    solr = RSolr.connect(:url => solr_url)
```

Comment:
You can make this bigger if you want but names is _millions_ of records so we can't post them all at once.
I plucked 1,000 out of thin air.